ivy/functional/ivy/elementwise.py

```diff
@@ -5092,6 +5092,223 @@ def subtract(
     ret
         an array containing the element-wise differences.
 
```

Comment:
Add lines referring to Array API standard: https://lets-unify.ai/ivy/deep_dive/11_docstrings.html
Also add container lines.